modules/js/planetaryFinal.js

```diff
@@ -76,129 +101,114 @@ function addPlanetaryGearComposite(centerX, centerY, constraintX, constraintY){
       toothHeight: toothHeight,
       numOfTeeth: steps,
       alternate: false,
-      lock: false
+      lock: false,
     })
-  )
+  );
 
   // add constraint to constraint array constraintArray[]
   constraintArray.push(
     // create constraint to rotate around
-    Constraint.create({pointA: { x: constraintX, y: constraintY },
+    Constraint.create({
+      pointA: { x: constraintX, y: constraintY },
       // body to constrain
-      bodyB: compositeArray[totalComposites-1].bodies[0], 
-      stiffness: 1
+      bodyB: compositeArray[totalComposites - 1].bodies[0],
+      stiffness: 1,
     })
-  )
+  );
   // add constraint to composite (composite to add to, constraint to add)
-  Composite.add(compositeArray[totalComposites-1], constraintArray[totalConstraints-1]);
-  Composite.add(compositeArray[totalComposites-1], Bodies.circle(centerX, centerY, 1))
+  Composite.add(
+    compositeArray[totalComposites - 1],
+    constraintArray[totalConstraints - 1]
+  );
+  Composite.add(
+    compositeArray[totalComposites - 1],
+    Bodies.circle(centerX, centerY, 1)
+  );
   // add composite to the world
-  World.add(engine.world,[compositeArray[totalComposites-1]] );
+  World.add(engine.world, [compositeArray[totalComposites - 1]]);
   compositeArray[1].isMotor = true;
   compositeArray[1].motorSpeed = 0.051;
   compositeArray[1].motorDir = -1;
 }
 //////////////////////////////////GEAR SIZES/////////////////////////////////////
-function smallGear1(){
+//small = 48
+//medium = 64
+//large = 80
+function gear1(rad) {
```

Comment:
These numerals in the function names are likely tied to something in the UI?  Probably better in the future to give them better names since "1" is hard to interpret when looking at the code.